### DISCUSSION TITLE
Portainer unsupported…what exactly does it mean?

### DISCUSSION THREAD

**Discussion by GizmoGoody** [alexbelgium/hassio-addons#626] [Q&A]
I run HA supervised, so installed the Portainer add-on to run Infinitude marked it as unsupported. I (sort of) understand why, but what exactly are the impacts to me using an unsupported version? Will I still be able to update HA core, integrations, and add-ons?

**Comment by alexbelgium** [comment]
Hi, here is an answer from the forum : https://community.home-assistant.io/t/you-are-running-an-unsupported-installation-what-does-it-mean/350715/2

**Reply by GizmoGoody** [reply]
Thanks, so I guess I'm good since I don't see the word "Unhealthy" anywhere?

Even if I wasn't getting updates, would I just need to disable the "offending" container temporarily to process updates, then re-enable the container OR would I need to uninstall the Portainer add-on entirely?

**Reply by alexbelgium** [reply]
Disabling the custom container is enough. Portainer by itself runs without any warnings

**Reply by GizmoGoody** [reply]
Thanks. I guess I'll do that until someone decides to create an Infinitude add-on or HA re-allows Portainer.

**Reply by GizmoGoody** [reply]
Just a quick update after 2023.1 was pushed out. It seems I'm getting all of the updates even while running the add-on. So, it doesn't appear there are any drawbacks.

**Reply by alexbelgium** [reply]
Thanks for the feedback
